src/_components/Navbar.jsx

```diff
@@ -1,29 +1,109 @@
 import React from "https://esm.sh/react";
 
-export default function Navbar() {
+// Global navigation links array
+const navigationLinks = [
+  { href: "/mission", text: "Mission" },
+  { href: "/projects", text: "Projects" },
+  { href: "/community", text: "Get Involved" },
+  { href: "/blog", text: "Blog" },
+];
+
+export default function NavigationBar() {
   const logo = "/assets/logos/logo_negative.png";
   return (
-    <nav className="flex items-center justify-around space-x-64 p-6 bg-primary z-50">
-      <a href="/" className="flex items-center text-white mr-60">
-        <img src={logo} alt="Logo" className="mr-2 w-10 h-10" />
-        <span className="self-center text-3xl font-semibold font-poppins whitespace-nowrap dark:text-white">
+    <nav className="relative flex items-center justify-between p-6 bg-primary z-50">
+      {/* Logo and Brand Name Section */}
```

Comment:
I don't know to what extent all the comments in the return statement make the code verbose. What do you think @Caposto @ShawnAviles?

Reply:
I agree, it is a little verbose. Since this is mostly JSX and not really any complicated logic, I don't think the comments are necessary. You had the right idea @ZhaoYuLL, but I think your code is self-explanatory and readable as.

Reply:
I removed the verbose comments